doc/Learn/Navigation/HowTo-DisplayItem.md

```diff
@@ -159,27 +162,28 @@ If you have a `ListView` that has items of different types, the navigation route
     }
     ```
 
-- Add a `ViewMap` and `RouteMap` for `ThirdPage` and `ThirdViewModel`, specifying the `AdvancedWidget` in the `DataMap`. Also update DataMap for `SecondViewModel` to be `BasicWidget` instead of `Widget`
+- Add a `DataViewMap` and `RouteMap` for `ThirdPage` and `ThirdViewModel`, specifying the `AdvancedWidget`. Also update `DataViewMap` for `SecondViewModel` to be `BasicWidget` instead of `Widget`
```

Comment:
```suggestion
- Add a `DataViewMap` and `RouteMap` for `ThirdPage` and `ThirdViewModel`, specifying the `AdvancedWidget`. Also, update `DataViewMap` for `SecondViewModel` to be `BasicWidget` instead of `Widget`
```